llmos-ai/llmos-operator
# ci: update webhook ci

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Need to add webhook CI.

**Solution:**
Added webhook image & CI configs.

**Related Issue:**

**Test plan:**
1. Run make install
2. Check if the webhook pod is running properly.
